README.md

```diff
@@ -59,6 +59,12 @@ Defaults to `null`
 
 Specify the cache control metadata value for all syncable objects 
 
+### `endpoint-url`
+
+Defaults to `null`
```

Comment:
Is there another way we can describe why this is optional, in the case of S3 is there a default, or is one even needed?